src/backend/opencl/CMakeLists.txt

```diff
@@ -108,12 +108,22 @@ INCLUDE_DIRECTORIES(${CLFFT_INCLUDE_DIRS})
 LINK_DIRECTORIES(${CLFFT_LIBRARY_DIR})
 
 OPTION(USE_SYSTEM_BOOST_COMPUTE "Use system BoostCompute" OFF)
-IF(USE_SYSTEM_BOOST_COMPUTE)
-    IF(Boost_VERSION VERSION_LESS "1.61")
+
+ADD_DEFINITIONS(-DBOOST_ALL_NO_LIB)
+SET(Boost_USE_STATIC_LIBS OFF)
+FIND_PACKAGE(Boost REQUIRED)
+
```

Comment:
We are not using anything else other than Boost.Compute in opencl backend. Is there any point i making this REQUIRED if we are not asking for 1.61?